packages/pluggableWidgets/badge-web/src/Badge.editorPreview.tsx

```diff
@@ -4,23 +4,24 @@ import { createElement, ReactElement } from "react";
 import { BadgePreviewProps } from "../typings/BadgeProps";
 import { Badge } from "./components/Badge";
 
-declare function require(name: string): string;
-
 export const preview = (props: BadgePreviewProps): ReactElement => {
-    const { class: classname, style, type, value, brandStyle, onClick } = props;
+    // TODO: Change PIW preview props typing (class -> className) generation to remove the ts-ignore below
+    // @ts-ignore
+    const { className, style, type, value, onClick } = props;
 
     return (
         <Badge
             type={type}
             value={value ? value : ""}
-            clickable={onClick != null}
-            brandStyle={brandStyle}
-            className={classname}
+            onClick={
+                onClick != null
```

Comment:
Editor preview is not clickable. So why not just `undefined`?

Reply:
Removed it.